yml/apdb.yaml

```diff
@@ -2140,7 +2140,6 @@ tables:
   - name: dipoleNdata
     "@id": "#DiaSource.dipoleNdata"
     datatype: int
-    nullable: false
     value: "0"
```

Comment:
Do we still want the value to be set to 0 here? None of the other entries have anything for "value," and in ip_diffim you seem to be setting `np.nan` in a bad fit situation. I just want to ensure this isn't an undesirable/vestigial situation.

Reply:
I removed the value: "0" here, since it gets set in ip_diffim anyways. We still need it, though, as ap_verify fails if it gets set to np.nan despite me allowing it to do so. Is it possible there is another place I need to edit?